app/page.tsx

```diff
@@ -57,6 +58,21 @@ export default function HomeScreen() {
         </Container>
       </Box>
 
+        {/* Banner Section */}
```

Comment:
The exact changes you have done here is not clear as you have replaced the entire banner section.

Reply:
it should change as contactNow Banner i have change it now